### PR TITLE
added new sim files; connect to ta1 servers

### PR DESCRIPTION
**Pull request by kaitlyn-sharo**
Jennifer looked through several CSVs to find some missing simulator data. I put what she found into json format so that it could be uploaded to production.

The alignment values had not been being calculated properly before, but now the code will connect to ta1 servers to get data. All urls that are subject to change are in the .env for easy access.

Additionally, before mongo would not update if an entry was already found, but now it will either insert, or if that fails, update

### CODE REVIEW

File: metrics-data/4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 .json
Comment (brianpippin) [inline metrics-data/4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 .json:1]: This file name has a " " before the ".json" not sure if it was intentional, but I had to add some code when I was working on a table to account for this.

Comment (nextcen-dgemoets) [reply metrics-data/4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 .json:1]: That is no doubt why I'm having trouble cloning the repo.  The error message I get is:
`error: invalid path 'metrics-data/4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 .json'`

Comment (brianpippin) [reply metrics-data/4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 .json:1]: Interesting that doesn't cause an issue on macs but windows chokes on it.

Comment (nextcen-dgemoets) [reply metrics-data/4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 .json:1]: Yep.  I think the problem is the trailing space in the *directory name*, not the json file.

Comment (kaitlyn-sharo) [reply metrics-data/4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 /4de6e23f-3c00-49d9-bbac-3a9febab4311_2024222 .json:1]: Totally missed these comments before. Main (with the change) should be merged in now.

File: .env [hunk 1]
```diff
@@ -1,3 +1,18 @@
 #### GENERAL CONFIG ####
 # level of logging (1=debug, 2=info, 3=warn, 4=error)
 LOG_LEVEL=2
+
+# location to access mongo database
+MONGO_URL="mongodb://simplemongousername:simplemongopassword@localhost:27030/?authSource=dashboard"
+
+# location of soartech server
+ST_URL="http://127.0.0.1:8084"
```
[the remark this answers is not in the document]
Comment (kaitlyn-sharo) [reply .env]: Done

**Review by brianpippin — APPROVED** [review]

File: .env [hunk 2]
```diff
@@ -1,3 +1,20 @@
 #### GENERAL CONFIG ####
 # level of logging (1=debug, 2=info, 3=warn, 4=error)
 LOG_LEVEL=2
+
+# location to access mongo database
+MONGO_URL="mongodb://simplemongousername:simplemongopassword@localhost:27030/?authSource=dashboard"
+
+# location of soartech server
+# ST_URL="http://127.0.0.1:8084" # development
+ST_URL="https://darpaitm.caci.com/soartech" # production
```
Comment (brianpippin) [inline .env:10]: This won't run with the " # production" on the same line it thinks it is part of the string.  You'll need to move the development and production call outs to the lines above the urls.

**Review by dereknop — APPROVED** [review]
Ingested both the raw json files to collection 'humanSimulatorRaw' and json files with alignment to 'humanSimulator'.

**Pull request merged**